stable/mongodb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mongodb
-version: 4.0.6
+version: 4.0.7
```

Comment:
Please bump the minor version. This is not simple 'patch'

Reply:
np done